src/main/views/cookies.njk

```diff
@@ -0,0 +1,213 @@
+            {{ t('This helps us count how many people visit the service by tracking if you’ve visited before') }}
+          </td>
+          <td class="text-area">{{ t('2 years') }}</td>
+        </tr>
+        <tr>
+          <td>_gat</td>
+          <td>
+            {{ t('Manages the rate at which page view requests are made') }}
+          </td>
+          <td class="text-area">{{ t('10 minutes') }}</td>
+        </tr>
+        <tr>
+          <td>_gid</td>
+          <td>
+            {{ t('Identifies you to the service') }}
+          </td>
+          <td class="text-area">{{ t('24 hours') }}</td>
+        </tr>
+        </tbody>
+      </table>
+
+      <h2 class="heading-small">{{ t('To turn our introductory message off') }} </h2>
+
+      <p>
+        {{ t('You may see a pop-up welcome message when you first visit the service. '
+        + 'We’ll store a cookie so that your computer knows you’ve seen it and knows not to show it again.' ) }}
+      </p>
+
+      <table class="form-group">
+        <thead>
+        <tr>
+          <th scope="col">Name</th>
+          <th scope="col">Purpose</th>
+          <th scope="col">Expires</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td>seen_cookie_message</td>
+          <td>
+            {{ t('Saves a message to let us know that you’ve seen our cookie message') }}
+          </td>
+          <td class="text-area">{{ t('1 month') }}</td>
+        </tr>
+        </tbody>
+      </table>
+
+      <h2
+        class="heading-small">{{ t('To store the answers you’ve given during your visit (known as a ‘session’)') }}
+      </h2>
+
+      <p>
+        {{ t('Session cookies are stored on your computer as you travel through a website, '
+        + 'and let the website know what you’ve seen and done so far. '
+        + 'These are temporary cookies and are automatically deleted a short while after you leave the website.') }}
+      </p>
+
+      <table class="form-group">
+        <thead>
+        <tr>
+          <th scope="col">Name</th>
+          <th scope="col">Purpose</th>
+          <th scope="col">Expires</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td>connect.sid</td>
+          <td>
+            {{ t('Carries details of your current session') }}
+          </td>
+          <td class="text-area">{{ t('When you close your browser') }}</td>
+        </tr>
+        <tr>
+          <td>sessionKey</td>
+          <td>
+            {{ t('Protects your session using encryption') }}
+          </td>
+          <td class="text-area">{{ t('When you close your browser') }}</td>
+        </tr>
+        </tbody>
+      </table>
+
+      <h2 class="heading-small">{{ t('To identify you when you come back to the service') }}</h2>
+      <p>{{ t('We use authentication cookies to identify you when you return to the service.') }}</p>
+
+      <table class="form-group">
+        <thead>
+        <tr>
+          <th scope="col">Name</th>
```

Comment:
I think these headers need to be translated (here and in other places in this file).